### PR TITLE
RELATED: RAIL-990 Table totals in AD

### PR DESCRIPTION
**Pull request by Hurtak**

### CODE REVIEW

File: docs/pivot_table_component.md [hunk 1]
```diff
@@ -173,6 +173,38 @@ const sortBy = [
...
+Alternatively, you can enable menu, through which user can turn on totals. For more information, see [Config](#config).
+
+![Pivot Table Menu Totals](assets/pivot_table_menu_totals.png "Pivot Table Menu Totals")
+
+## Config
+
+The following settings can be configured:
+
+- Separators used when formatting numbers. See [Separator Number Format](chart_config.html#change-a-separator-in-the-number-format).
+- Menu items. Currently, we only support totals section in the menu.
```
Comment (no23reason) [inline docs/pivot_table_component.md]: Maybe
> Currently, only totals section is supported.

File: docs/pivot_table_component.md [hunk 2]
```diff
@@ -183,7 +215,7 @@ You can display rows with aggregated measure data using the `totals` prop. For m
 | columns | false | [Attribute[]](afm.md#attribute) | An array of attribute definitions that breaks measure data into columns (either measures, or rows, or columns must be provided for the pivot table to render properly) |
 | totals | false | [Total[]](table_totals.md) | An array of total definitions |
 | filters | false | [Filter[]](filter_visual_components.md) | An array of filter definitions |
-| config | false | [ChartConfig](chart_config.md) | The configuration object |
```
Comment (no23reason) [inline docs/pivot_table_component.md]: Is Chart config no longer relevant?

Comment (Hurtak) [reply docs/pivot_table_component.md]: `ChartConfig` was wrong, now we describe correct interface

File: docs/pivot_table_component.md (hunk 1 above)
Comment (Hurtak) [reply docs/pivot_table_component.md]: updated to `Currently, only totlas section is supported.`

File: docs/pivot_table_component.md [hunk 3]
```diff
@@ -173,6 +173,38 @@ const sortBy = [
...
+Alternatively, you can enable menu, through which user can turn on totals. For more information, see [Config](#config).
+
+![Pivot Table Menu Totals](assets/pivot_table_menu_totals.png "Pivot Table Menu Totals")
+
+## Config
+
+The following settings can be configured:
+
+- Separators used when formatting numbers. See [Separator Number Format](chart_config.html#change-a-separator-in-the-number-format).
+- Menu items. Currently, only totlas section is supported.
```
Comment (no23reason) [inline docs/pivot_table_component.md]: Typo totlas -> totals

Comment (Hurtak) [reply docs/pivot_table_component.md]: Fixed

**Comment by no23reason** [comment]
LGTM

**Pull request merged**